include/nuttx/clock.h

```diff
@@ -711,6 +711,7 @@ clock_t clock_systime_ticks(void);
  ****************************************************************************/
 
 int clock_systime_timespec(FAR struct timespec *ts);
+int clock_systime_timespec_rtc_nolock(FAR struct timespec *ts);
```

Comment:
could we remove this special api from common code?

Reply:
done